src/contexts/UserIdContext.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from 'react';
+
+export const UserIdContext = React.createContext(6303);
+
+type Props = {
+  children: ReactNode;
+};
+
+export const UserIdProvider: React.FC<Props> = ({ children }) => {
+  return (
+    <UserIdContext.Provider value={6303}>
```

Comment:
What is 6303 mean? Don't hardcode the user, please. There should be the login form first 
<img width="789" alt="image" src="https://user-images.githubusercontent.com/99403198/219852894-fe9d3cf6-a953-4a60-9363-c59cbccad81a.png">